servers/lumen/app/Http/Controllers/Phq9Controller.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace App\Http\Controllers;
+use App\Phq9;
+use Illuminate\Http\Request;
+use Ramsey\Uuid\Uuid;
+use DB;
+use Predis\Autoloader;
+\Predis\Autoloader::register();
+
+
+class Phq9Controller extends Controller
+{
+    public function  showAllPhq9Screen() 
+    {
+        $phq9Data = DB::select(DB::raw('SELECT serverSideRowUuid, ptUUID as patientUUID, questionUUID as phq9QuestionMasterId,  answerValue as optionValueChosenByPatient,  round(UNIX_TIMESTAMP(ROW_START) * 1000) as ROW_START, round(UNIX_TIMESTAMP(ROW_END) * 1000) as ROW_END, UNIX_TIMESTAMP(submittedOnDateTimeInMilliseconds) * 1000 as submittedOnDateTimeInMilliseconds FROM sc_scr.phq9PtResponses FOR SYSTEM_TIME ALL order by ROW_START desc'));
+        return response()->json($phq9Data);
+    }
+
+    public function showOnePhq9Screen($id) 
+    {
+        return response()->json(Phq9::find($id));
+    }
+
+    public function create(Request $request) 
+    {
+        $requestData = $request->all();
+
+        $phq9Data = array(
+            'serverSideRowUuid' => $requestData['data']['serverSideRowUuid'],
+            'ptUUID' => $requestData['data']['ptUUID'],
+            'questionUUID' => $requestData['data']['questionUUID'],
+            'answerValue' => $requestData['data']['answerValue'],
+            'submittedOnDateTimeInMilliseconds' => $requestData['data']['submittedOnDateTimeInMilliseconds'],
```

Comment:
How to insert 'submittedOnDateTimeInMilliseconds' in DB, field type datetime(3) without using 'FROM_UNIXTIME'?